staging/src/k8s.io/client-go/openapi/groupversion.go

```diff
@@ -39,21 +37,15 @@ func newGroupVersion(client *client, item handler3.OpenAPIV3DiscoveryGroupVersio
 	return &groupversion{client: client, item: item}
 }
 
-func (g *groupversion) Schema() (*openapi_v3.Document, error) {
+func (g *groupversion) Schema(contentType string) ([]byte, error) {
 	data, err := g.client.restClient.Get().
```

Comment:
Why don't you return the result of that function directly?

Reply:
changed